app/assets/src/components/views/projects/ProjectsView.jsx

```diff
@@ -21,6 +25,7 @@ class ProjectsView extends React.Component {
 
     this.columns = [
       {
+        columnData: PROJECT_TABLE_COLUMNS["project"],
```

Comment:
For DRY'ing this out, you could instead have a line looping through `this.columns` and adding `columnData: PROJECT_TABLE_COLUMNS[dataKey]`